apps/web/src/features/search/hooks/use-search-results.ts

```
@@ -0,0 +1,58 @@
+import { useQuery, useMutation } from '@tanstack/react-query';
+import { convexQuery, useConvexMutation } from '@convex-dev/react-query';
+import { api } from '@vibechecc/convex';
+import type { SearchRequest, SearchResponse } from '@vibechecc/types';
```

ESLint

[error] 'SearchResponse' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)